vcd/resource_vcd_vm_affinity_rule_test.go

```diff
@@ -389,7 +389,7 @@ func testAccCheckVmAffinityRuleDestroy(rule *govcd.VmAffinityRule, orgName, vdcN
 // makeEmptyVapp creates a given vApp without any VM
 func makeEmptyVapp(vdc *govcd.Vdc, name string) (*govcd.VApp, error) {
 
-	err := vdc.ComposeRawVApp(name)
+	err := vdc.ComposeRawVApp(name, name)
```

Comment:
Shouldn't the second parameter be `description`?

Reply:
changed the function signature to include an explicit description